src/js/components/TaskListItemComponent.jsx

```diff
@@ -124,7 +125,7 @@ var TaskListItemComponent = React.createClass({
         <td>
             <a href={taskUri}>{taskId}</a>
           <br />
-          {buildTaskAnchors(task)}
+          {buildTaskAnchors(task)} <TaskMesosUrlComponent task={task} text="mesos details"/>
```

Comment:
I would prefer to remove this here and only stick with the link in the TaskDetailComponent.
What do you mean?